cli/ops/repl.rs

```diff
@@ -11,15 +12,12 @@ use deno::*;
 use std::sync::Arc;
 use std::sync::Mutex;
 
-pub fn init(i: &mut Isolate, s: &ThreadSafeState) {
+pub fn init(i: &mut Isolate, gs: &ThreadSafeGlobalState, s: &ThreadSafeState) {
```

Comment:
If the state is global, why does it need to be tediously passed around? A static global should be used to reference it.

Reply:
Probably it doesn't - I'll rework to use static global